src/content/api/account-memberships.mdx

```diff
@@ -0,0 +1,272 @@
+
+  ### Attributes
+  <Properties>
+    <Property name="userId" type="string" required>
+      The id of the user the Membership belongs to.
+    </Property>
+
+    <Property name="role" type="string" required>
+      The role governing Membership permissions.
+    </Property>
+  </Properties>
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/accounts/Jaim1Cu1Q55uooxSens6yk/members \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "userId": "da75ad90-9a5b-4df0-8374-f48b3a8fbfcc",
+        "role": "account-owner"
+      }'
+    ```
+
+    ```json
+    {
+      "accountId": "Jaim1Cu1Q55uooxSens6yk",
+      "accountType": "org",
+      "userId": "da75ad90-9a5b-4df0-8374-f48b3a8fbfcc",
+      "role": "account-owner",
+      "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+      "createdAt": "2020-06-12T01:17:46.499Z",
+      "modifiedAt": "2020-06-12T01:17:46.499Z",
+      "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey"
+    }
+    ```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/accounts/{accountId}/members"
+>
+  ## List members
+
+  This endpoint allows you to retrieve the list memberships to an account.
+
+  ### Attributes
+  No attributes.
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl https://service.centrapay.com/api/accounts/Jaim1Cu1Q55uooxSens6yk/members \
+      -H "X-Api-Key: $api_key"
+    ```
+
+    ```json
+    [
+      {
+        "accountName": "Centrapay Cafe",
+        "accountType": "org",
+        "accountId": "Jaim1Cu1Q55uooxSens6yk",
+        "userId": "b657195e-dc2f-11ea-8566-e7710d592c99",
+        "createdAt": "2020-06-01T21:57:25.888Z",
+        "role": "account-owner",
+        "firstName": "John",
+        "lastName": "Doe",
+        "email": "john.doe@centrapay.com"
+      },
+      {
+        "accountName": "Centrapay Tea Warehouse",
+        "accountType": "org",
+        "accountId": "0f9nvqdcn5eaaDLefkg1Xt",
+        "userId": "9f4b3bae-dc30-11ea-ab70-2743d9be3dd5",
+        "createdAt": "2020-06-02T10:l4:33.021Z",
+        "role": "account-owner",
+        "firstName": "Jane",
+        "lastName": "Doe",
+        "email": "jane.doe@centrapay.com"
+      }
+    ]
+    ```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="DELETE"
+  path="/api/accounts/{accountId}/members/{userId}"
+>
+  ## Revoke Member
+
+  This endpoint allows you to revoke a users membership to an account.
+
+  ### Attributes
+  No attributes.
+
+  ### Errors
+  | Status |           Code           |                          Description                           |
+  | :----- | :----------------------- | :------------------------------------------------------------- |
+  | 403    | LAST_OWNER_NOT_REVOKABLE | The last remaining membership to an account cannot be revoked. |
+
+  <CodeGroup slot="code-examples">
```

Comment:
@AndyClifford what does this return? Even if it returns an empty object it could be nice to always document a response.

Reply:
It returns nothing, not even an empty object 😅 but I agree some sort of documentation would be nice - do you have any suggestions?